manifests/iscsi-demo-target.yaml.in

```diff
@@ -3,6 +3,7 @@ kind: PersistentVolumeClaim
 metadata:
   name: disk-custom
 spec:
+  storageClassName: ""
```

Comment:
Why not use a real name like "kubevirt-iscsi-demo"?

Reply:
Good question. If you are working on this anyway, then I can close this request and we move fwd with your code.